### PR TITLE
RATIS-1600. Add protobuf-java-util

### PR DESCRIPTION
**Pull request by tisonkun**
Signed-off-by: tison <wander4096@gmail.com>

cc @szetszwo

### CODE REVIEW

File: misc/pom.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
   <parent>
     <groupId>org.apache.ratis</groupId>
     <artifactId>ratis-thirdparty</artifactId>
-    <version>1.0.1</version>
+    <version>1.1.0-SNAPSHOT</version>
```
Comment (tisonkun) [inline misc/pom.xml:20]: Bump version to verify locally. I'm unsure whether it's proper to be included in this patch.

Comment (szetszwo) [reply misc/pom.xml:20]: @tisonkun , thanks a lot for working on this!  Please don't change the version. We will let the release managers to change it.  The change looks good other than that.

Comment (tisonkun) [reply misc/pom.xml]: @szetszwo OK. Updated. Please give it another pass :)

**Review by szetszwo — APPROVED** [review]
+1 the change looks good.

**Pull request merged**